content/using.html

```diff
@@ -66,25 +66,28 @@
 
   {% endcall %}
 
-  {% call subsection('Channels')%}
+  {% call subsection('Labels')%}
 
-  Each file within a package may be tagged in one or more channels. A file may also not be tagged in any channels.
-  One common use of channels is to divide a package's files into development, testing, and main channels called 
-  dev, test, and main. This allows package authors to upload files for development or testing purposes without
-  affecting non-development users.
-
-  At [https://**pypi**.anaconda.org/conda](https://pypi.anaconda.org/conda) a channel
-  dropdown lists all the available channels in this user namespace.
+  Each file within a package may be tagged with one or more labels, or not tagged at all. 
+  Labels are commonly used to divide a package's files into sections labeled development, 
+  testing (and/or other custom labels), and main or dev, test, main or other custom label. 
+  The use of labels allows package authors to upload files for development or testing 
+  purposes without affecting non-development users. 
+  
+  Packages that are uploaded are labeled "main" by default. The version labeled main 
+  is also delivered by default unless a user specifies a different label. 
+  
+  At [https://**pypi**.anaconda.org/conda](https://pypi.anaconda.org/conda) a dropdown 
+  menu lists all the available labels.
 
-  For example all of the pypi packages tagged as 'dev' can be shown here:
+  For example, all of the pypi packages labeled as 'dev' can be shown here:
 
-   *  [https://pypi.anaconda.org/conda/channel/dev](https://pypi.anaconda.org/conda/channel/dev)
+   *  [https://pypi.anaconda.org/conda/label/dev](https://pypi.anaconda.org/conda/label/dev)
 
-  The default channel is the **main** channel.
-  If a file is tagged in the **main** channel then the channel name may be omitted from the URL.
+  The default label is **main**. If a file is labeled as **main** then the label name may be omitted from the URL.
   For example the following repositories are equivalent:
 
-   *  [https://**pypi**.anaconda.org/conda/channel/main](https://pypi.anaconda.org/conda/channel/main)
+   *  [https://**pypi**.anaconda.org/conda/label/main](https://pypi.anaconda.org/conda/label/main)
```

Comment:
This is not a real url. I'm not sure if you want to change it to https://**conda**.anaconda.org/anaconda